website/docs/docs/building-a-dbt-project/building-models/configuring-incremental-models.md

```diff
@@ -64,16 +67,44 @@ from raw_app_data.events
+A `unique_key` determines whether a record has new values and should be updated. By using `unique_key`, you can ensure that each row from the source table is represented by a single row in your incremental model, without duplicates. Not specifying a `unique_key` will result in append-only behavior, which means dbt inserts all rows returned by the model's SQL into the preexisting target table without regard for whether the rows represent duplicates.
+
+<VersionBlock firstVersion="0.20" lastVersion="1.0">
+
+This optional parameter for incremental models specifies a field that can uniquely identify each row within your model. You can define `unique_key` in a configuration block at the top of your model. If your model doesn't contain a single field that is unique, you should consider creating `unique_key` in your model definition.
+
+</VersionBlock>
+
+<VersionBlock firstVersion="1.1">
+
+This optional parameter for incremental models specifies a field (or combination of fields) that can uniquely identify each row within your model. You can define `unique_key` in a configuration block at the top of your model, and it can be a list in addition to a single column name.
+
+The `unique_key` should be supplied in your model definition as a string representing a simple column or a list of single quoted column names that can be used together, for example, `[‘col1, ‘col2’, …])`. We recommend you create a list of columns in your model.
 
-As an example, consider a model that calculates the number of daily active users (DAUs), based on an event stream. As source data arrives, you will want to recalculate the number of DAUs for both the day that dbt last ran, and any days since then. The model would look as follows:
+:::tip
+We recommend using universally supported syntax to avoid problems and adhere to the same standards. For example, we recommend you pass these columns as a list: `unique_key = ['user_id', 'session_number']` instead of an expression: `concat(user_id, session_number)`.
+:::
+
+</VersionBlock>
+
+When using a `unique_key` for a given row in a `source` table:
+
+* If all `unique_key` identified columns are present in the target table, an update procedure likely takes place, depending on the incremental strategy. For more information, see [About incremental_strategy](#about-incremental_strategy).
+* If any one of the `unique_key` columns in a source row does not find a matching target table row, the entire row is inserted into the target table.
+
+:::info
+While common incremental strategies, such as`delete+insert` + `merge`, might use `unique_key`, others don't. For example, the `insert_overwrite` strategy does not use `unique_key`. For more information, see [About incremental_strategy](#about-incremental_strategy).
```

Comment:
👍 love this, I think we can give just a tiny bit more color:
```suggestion
While common incremental strategies, such as`delete+insert` + `merge`, might use `unique_key`, others don't. For example, the `insert_overwrite` strategy does not use `unique_key`, because it operates on partitions of data rather than individual rows. For more information, see [About incremental_strategy](#about-incremental_strategy).
```